### ISSUE TITLE
Log noise: INFO level is swamped!

### ISSUE THREAD

**Issue by decibelcooper** [Comcast/rulio#57]
Hi Maintainers,

I would very much like to find a way to quiet down the INFO log level while keeping some useful logs in there.  I see some extreme offenders that tend to spam this log level for no apparent reason.  A good example would be this line:

https://github.com/Comcast/rulio/blob/master/sys/system.go#L101

I am wondering if you would be receptive to a PR that moves all of these "hi, the function is starting" logs in `sys/system.go` to the DEBUG level?

**Comment by jsccast** [comment]
Yeah, that change sounds good. Historically Rulio has emitted too many `INFO` messages.
